src/plugins/intel_cpu/src/nodes/convert.h

```diff
@@ -40,7 +42,7 @@ class Convert : public Node {
     const MemoryDesc& getInput() const { return *input; }
     const MemoryDesc& getOutput() const { return *output; }
 
-    bool needPrepareParams() const override { return false; }
+    bool needPrepareParams() const override { return true; }
```

Comment:
Do we really need to do it unconditionally even if the shape hasn't changed?

Reply:
fixed